packages/server/src/server/auth/handlers/authorize.ts

```diff
@@ -128,7 +130,7 @@ export function authorizationHandler({ provider, rateLimit: rateLimitConfig }: A
                 {
                     state,
                     scopes: requestedScopes,
-                    redirectUri: redirect_uri,
+                    redirectUri: redirect_uri!, // TODO: Someone to look at. Strict tsconfig showed this could be undefined, while the return type is string.
                     codeChallenge: code_challenge,
```

Comment:
n00b question on this comment: all these added `!` we're doing in this PR, are these indicating functional changes or because we tightened up type checking we're actually discovering potential gaps?

Reply:
Already answered elsewhere.